Assets/Code/Sprite/PositionGrid/PositionGrid.cs

```diff
@@ -32,14 +32,37 @@ private void AssignTransformPositionsToTiles()
             for (int j = 0; j < yBound; j++)
             {
                 Vector3 tilePosition = new Vector3(xTransform, yTransform);
-                this.grid[i,j] = Instantiate(tilePrefab, tilePosition, Quaternion.identity, tileContainer);
+                grid[i,j] = Instantiate(tilePrefab, tilePosition, Quaternion.identity, tileContainer);
                 yTransform += yOffset;
             }
 
             xTransform += xOffset;
         }
     }
 
+    public void AssignOwnershipOfTiles()
+    {
+        Player player = GameObject.FindWithTag("Player").GetComponent(typeof(Player)) as Player;
+        Enemy enemy = GameObject.FindWithTag("Enemy").GetComponent(typeof(Enemy)) as Enemy;
+
+        //Assign the left half to the Player
+        for (int i = 0; i < (xBound/2); i++)
+        {
+            for (int j = 0; j < yBound; j++)
+            {
+                grid[i,j].Owner = player;
+            }
+        }
+        //Assign the right half to the Enemy
+        for (int i = (xBound/2); i < xBound; i++)
+        {
+            for (int j = 0; j < yBound; j++)
+            {
+                grid[i,j].Owner = enemy;
+            }
+        }
+    }
+
```

Comment:
I think this would be easier to do by just giving each tile an enumerated "I belong to this team" variable. This keeps the tiles from having to store specific characters as valid owners, and would allow you to easily set up a grid with multiple enemies present (or multiple allied characters, should that ever occur).

Reply:
Agreed. It will easier to scale that way.